README-SW.md

```diff
@@ -57,39 +57,39 @@ Ikiwa una maswali au maoni, tafadhali jenga suala.
 
 ## Kuchangia kwa Open Source kwa ujumla
 
-- [Open Source Guide](https://opensource.guide/) - Mwongozo wa hatua kwa hatua wa jinsi ya kuchangia kwenye miradi ya Open Source.
-- [First Timers Only](https://www.firsttimersonly.com/) - Rasilimali kwa watu wanaochangia kwa mara ya kwanza kwenye miradi ya Open Source.
-- [Up For Grabs](https://up-for-grabs.net/) - Orodha ya miradi ya Open Source inayotafuta mchango.
+- [Mwongozo wa Open Source](https://opensource.guide/) - Mwongozo wa hatua kwa hatua wa jinsi ya kuchangia kwenye miradi ya Open Source.
```

Comment:
**Option**
Open Source can also be translated to `Chanzo Huru` au `Chanzo Wazi`